packages/web/src/components/pool/pool-pair-info-content/PoolPairInfoContent.tsx

```diff
@@ -227,7 +231,11 @@ const PoolPairInfoContent: React.FC<PoolPairInfoContentProps> = ({
                     className="image-logo"
                   />
                   <span>
-                    {convertToKMB(`${tokenBBalance}`)}{" "}
+                    {formatTokenExchangeRate(`${tokenBBalance}`, {
+                      minLimit: 0.000001,
+                      maxSignificantDigits: 6,
+                      fixedDecimalDigits: 6,
+                    })}{" "}
```

Comment:
Get the value from the token's meta information.

Reply:
Is that right? i remember that the requirement from Dongwon is show 6 decimals (fixed).

Reply:
I double-checked, and that's what was processed.
I'm going to resolve that.